tests/test_api_call.py

```diff
@@ -0,0 +1,50 @@
+import pytest
+import requests_mock
+
+from APICall import *
+
+key = os.environ['FRED_API_TOKEN_REGULATIONS_GOV']
+base_url = 'https://api.data.gov:443/regulations/v3/documents.json?'
+
+@pytest.fixture
+def mock_req():
+    with requests_mock.Mocker() as m:
+        yield m
+
+def test_exception_thrown(mock_req):
+    mock_req.get(add_api_key(base_url),status_code=300)
+    with pytest.raises(TemporaryException):
+        call(add_api_key(base_url))
+
+def test_happy_path(mock_req):
```

Comment:
Make this the first test to show what works and then followed by how it could fail